src/app/story/storyrow/totaltimes/totaltimes.component.css

```diff
@@ -0,0 +1,27 @@
+.timeline-wrapper {
+    display: none !important;
```

Comment:
Remove the `!important` or else it keeps the media query below from taking effect.